static/gsAdmin/components/changePlanAction.spec.tsx

```diff
@@ -162,27 +162,20 @@ describe('ChangePlanAction', () => {
 
     // Wait for async data to load
     await waitFor(() => {
-      expect(screen.getByTestId('am3-tier')).toBeInTheDocument();
+      expect(screen.getByRole('tab', {name: 'AM3'})).toBeInTheDocument();
     });
 
     // Verify the tabs are rendered
-    expect(screen.getByTestId('am3-tier')).toBeInTheDocument();
-    expect(screen.getByTestId('am2-tier')).toBeInTheDocument();
-    expect(screen.getByTestId('mm2-tier')).toBeInTheDocument();
+    expect(screen.getByRole('tab', {name: 'AM3'})).toBeInTheDocument();
+    expect(screen.getByRole('tab', {name: 'AM2'})).toBeInTheDocument();
+    expect(screen.getByRole('tab', {name: 'MM2'})).toBeInTheDocument();
 
     // Verify at least one plan option is displayed
     expect(screen.getByTestId('change-plan-radio-btn-am3_business')).toBeInTheDocument();
 
     // Test basic interaction - click on AM2 tier
-    const am2Tab = screen.getByTestId('am2-tier');
+    const am2Tab = screen.getByRole('tab', {name: 'AM2'});
     await userEvent.click(am2Tab);
-
-    // Verify AM2 tier becomes active
-    await waitFor(() => {
```

Comment:
This test is basically just testing that clicking a tab sets it to active, which doesn't seem super valuable in the context of changePlanAction. Could we instead assert that the correct tab is rendered here?